umbraco/Umbraco-CMS
# v9: Removed obsolete SystemFiles class

# Notes
* Deleted obsolete SystemFiles class

# How to test
* Build and see if there are errors
* Run Unit tests and see if there are errors
